gateway/modules/operations.go

```diff
@@ -75,6 +77,32 @@ func (m *Modules) SetProjectConfig(c *config.Project) error {
 	return nil
 }
 
+// CloseProjectConfig close the config all modules
+func (m *Modules) CloseProjectConfig(c *config.Config, le *letsencrypt.LetsEncrypt, ingressRouting *routing.Routing) {
```

Comment:
The first parameter should be project config, i believe the other 2 parameters are not required